src/blas/backends/mklgpu/mkl_blas_sycl_buffer.cpp

```diff
@@ -32,174 +33,188 @@ void gemm(cl::sycl::queue &queue, oneapi::mkl::transpose transa, oneapi::mkl::tr
           std::int64_t m, std::int64_t n, std::int64_t k, float alpha,
           cl::sycl::buffer<float, 1> &a, std::int64_t lda, cl::sycl::buffer<float, 1> &b,
           std::int64_t ldb, float beta, cl::sycl::buffer<float, 1> &c, std::int64_t ldc) {
-    ::mkl::gpu::sgemm(queue, MKL_COL_MAJOR, ::mkl::cblas_convert(transa),
-                      ::mkl::cblas_convert(transb), m, n, k, alpha, a, lda, b, ldb, beta, c, ldc);
+    ::oneapi::mkl::gpu::sgemm(queue, MKL_COL_MAJOR, ::mkl::cblas_convert(transa),
+                              ::mkl::cblas_convert(transb), m, n, k, alpha, a, lda, b, ldb, beta, c,
+                              ldc);
 }
 
 void gemm(cl::sycl::queue &queue, oneapi::mkl::transpose transa, oneapi::mkl::transpose transb,
           std::int64_t m, std::int64_t n, std::int64_t k, double alpha,
           cl::sycl::buffer<double, 1> &a, std::int64_t lda, cl::sycl::buffer<double, 1> &b,
           std::int64_t ldb, double beta, cl::sycl::buffer<double, 1> &c, std::int64_t ldc) {
-    ::mkl::gpu::dgemm(queue, MKL_COL_MAJOR, ::mkl::cblas_convert(transa),
-                      ::mkl::cblas_convert(transb), m, n, k, alpha, a, lda, b, ldb, beta, c, ldc);
+    ::oneapi::mkl::gpu::dgemm(queue, MKL_COL_MAJOR, ::mkl::cblas_convert(transa),
```

Comment:
As far as I know "::" was added because opensource had oneapi namespace and Intel oneMKL beta08 didn't have it yet. 

So for beta09  `oneapi::mkl::gpu::dgemm` should work fine.